src/network/bmproto.py

```diff
@@ -269,16 +296,19 @@ def decode_simple(self, char="v"):
             raise BMProtoInsufficientDataError()
 
     def bm_command_error(self):
+        """Decode an error message and log it"""
+        # pylint: disable=unused-variable
         fatalStatus, banTime, inventoryVector, errorText = self.decode_payload_content("vvlsls")
         logger.error("%s:%i error: %i, %s", self.destination.host, self.destination.port, fatalStatus, errorText)
         return True
 
     def bm_command_getdata(self):
+        """Decode object data, conditionally append a newly created object to the write buffer"""
```

Comment:
```suggestion
        """Incoming request for object(s), if we have them and some other conditions are fulfilled, append them to the write queue."""
```

PS. the wire protocol works very similarly to Bitcoin, so even if the command isn't documented in Bitmessage Wiki, just look at how it works in Bitcoin.